Dockerfile

```diff
@@ -2,9 +2,8 @@ FROM ubuntu:bionic
 
 RUN apt-get update && apt-get install -qy build-essential unixodbc-dev
 
-COPY . build/
-
-RUN cd build && make
+COPY . .
+RUN make
```

Comment:
I believe this splats the contents of this repository into `/`, which is pretty weird. I think this should stay as-is, and the path below should be updated to `/build/build/chBenchmark`. Or change this copy to copy into `/workdir` instead, if the `/build/build` bothers you, so that you have `/workdir/build/chBenchmark`.